metaflow/metaflow_environment.py

```diff
@@ -13,11 +13,11 @@
 
 
 class InvalidEnvironmentException(MetaflowException):
-    headline = 'Incompatible environment'
+    headline = "Incompatible environment"
```

Comment:
can you please remove the `black` formatting changes to this file?
so we can review only what was changed?

Reply:
same with `metaflow/mflog/__init__.py`

Reply:
Yup, will do.

Reply:
Agree with removing the formatting.

I similarly have an urge to format core metaflow code, but the main goal is to have less merge conflict when pulling netflix upstream.

Reply:
Removed.